src/main/drivers/timer.h

```diff
@@ -104,6 +104,12 @@ typedef struct timerHardware_s {
     DMA_Channel_TypeDef *dmaRef;
 #endif
     uint8_t dmaIrqHandler;
+#if defined(STM32F7)
```

Comment:
Should this be dependent on `STM32F7`, or on `USE_DSHOT_DMAR`? It seems to be only used for `USE_DSHOT_DMAR`.